src/babel/transformation/transformers/minification/remove-react-prop-types.js

```diff
@@ -0,0 +1,14 @@
+import * as t from "../../../types";
+
+export var metadata = {
+  optional: true,
+  group: "builtin-setup"
+};
+
+export var Property = {
+  exit(node) {
+    if (node.key.name === 'propTypes') {
```

Comment:
While unlikely to come up, this will also remove:

```javascript
var obj = {
  [propTypes]: {}
};
```

It'd be best to replace this with:

```javascript
!node.computed && this.get("key").matchesPattern("propTypes")
```